lib/src/layer.dart

```diff
@@ -75,7 +75,13 @@ abstract class Layer {
 
   /// Hex-formatted tint color (#RRGGBB or #AARRGGBB) that is multiplied with
   /// any graphics drawn by this layer or any child layers (optional).
-  String? tintColor;
+  String? tintColorHex;
```

Comment:
Is this really needed, if the user wants to get the hex code for a color it should be possible to get it from a `Color` object?

Reply:
There is no clean way to go from `ui.Color` to a hex string, you have to do something like:

```dart
final hexString = "0x${color.value.toRadixString(16).padLeft(8, '0')}"
```

I think it makes sense to keep exposing the hex string to clients in case they want to integrate with a different API (especially a web client). Also, it seems in line with the vision of this library to be a thin wrapper over the parsed `.tmx` file, a client may want to access the raw version of the value from the file.

If you think it bloats the API though, I'm fine with removing the hex string properties.

Reply:
Alright, let's keep this then, hopefully we won't have more fields that need to be duplicated later. 😅